pkg/config/runtime/runtime_http.go

```diff
@@ -71,6 +71,8 @@ func unique(src []string) []string {
 // RouterInfo holds information about a currently running HTTP router.
 type RouterInfo struct {
 	*dynamic.Router // dynamic configuration
+	// EffectivePriority is the user set priority if defined, or the rule's computed priority otherwise.
+	EffectivePriority int `json:"effectivePriority,omitempty"`
```

Comment:
I don't see any advantage to have `EffectivePriority` and `Priority` altogether, I'm fine to override `Priority` with the `effectivePriority`. Therefore, no need to change integration tests rawdata assets